eharris93/ELB
# [P33] Fortress Functionality

https://trello.com/c/J9KiKIDD

This PR adds the logic to assign lost immortals to random fortresses and also the logic to know when a battle should start with a lost immortal.

VERIFICATION==
1. Play around the board and verify that there are 4 fortresses that say "There is a chill in the air" for each player. This differs when the game is reloaded.
2. You can reload the game and find one fortress for one player which does this for the following test:
3. Go back and forth from the fortress capturing surrounding tiles.
4. Verify that after three surrounding tiles are captured, the boss appears.
5. verify that nothing appears when you go back  to the fortress afterwards
